### PR TITLE
[system-probe] add missing namespaces to config command

### PR DESCRIPTION
**Pull request by paulsemel**
### What does this PR do?

Adds potential submodules configuration namespaces to the `system-probe config` command. As for now, only the `system_probe_config` is displayed.

### Motivation

This makes it easier to know what is configurable and what is not.

### Describe how to test/QA your changes

The config subcommands are working correctly locally. The old behaviour of `getFullConfig` has been maintained (which is, when only one namespace is provided, it doesn't show itself at the root level of the yaml output).

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: cmd/system-probe/api/config.go [hunk 1]
```diff
@@ -7,14 +7,25 @@ package api
...
 // setupConfigHandlers adds the specific handlers for /config endpoints
 func setupConfigHandlers(r *mux.Router) {
-	r.HandleFunc("/config", settingshttp.Server.GetFull(config.Namespace)).Methods("GET")
+	r.HandleFunc("/config", settingshttp.Server.GetFull(getAggregattedNamespaces()...)).Methods("GET")
 	r.HandleFunc("/config/list-runtime", settingshttp.Server.ListConfigurable).Methods("GET")
 	r.HandleFunc("/config/{setting}", settingshttp.Server.GetValue).Methods("GET")
 	r.HandleFunc("/config/{setting}", settingshttp.Server.SetValue).Methods("POST")
 }
+
+func getAggregattedNamespaces() []string {
```
Comment (brycekahle) [inline cmd/system-probe/api/config.go:23]: typo: `getAggregatedNamespaces()`

File: cmd/system-probe/api/config.go [hunk 2]
```diff
@@ -7,14 +7,25 @@ package api
...
+	namespaces := []string{
+		config.Namespace,
+	}
+	for _, m := range modules.All {
+		namespaces = append(namespaces, m.ConfigNamespaces...)
```
Comment (brycekahle) [inline cmd/system-probe/api/config.go:28]: 👍

**Review by brycekahle — APPROVED** [review]
Fix the one typo, but otherwise looks good.

**Review by djmitche — COMMENTED** [review]
`pkg/config/settings/http/server.go` 👍🏽

**Review by djmitche — APPROVED** [review]

**Pull request merged**